test/northstar-client.test.js

```diff
@@ -39,15 +41,30 @@ test('NorthstarClient should respond to Users', () => {
 test('NorthstarClient.Users.create should return an object', async (t) => {
   const timestamp = Date.now();
   const email = `test+northstar-js+${timestamp}@${config.testCreateEmailDomain}`;
+  const source = config.testCreateSource;
+
   const data = {
     email,
+    source,
     password: `password+${timestamp}`,
-    source: config.testCreateSource,
   };
 
   const client = new NorthstarClient(config);
   const user = await client.Users.create(data);
   user.should.have.property('id');
   t.deepEqual(email, user.email);
-  t.deepEqual(config.testCreateSource, user.source);
+  t.deepEqual(source, user.source);
+  // Set our userId for next test.
+  userId = user.id;
```

Comment:
I know we are playing with 🔥 already, but isn't this a little more dangerous? I think ava tests are asynchronous within the file. There could potentially be a chance of trying to update an user that is not created yet.

Reply:
I've updated this branch to instead test Updating Users by passing a config value for a Northstar ID (defaulting to the [test value hardcoded in 1.0.0](https://github.com/DoSomething/northstar-js/blob/1.0.0/test/northstar-client.test.js#L40)). Once the Gambit Conversations <> Northstar work is complete, let's revisit test coverage using Nock (#25).